queries.sql

```diff
@@ -0,0 +1,55 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * from animals WHERE name like '%mon';
+SELECT name from animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT name from animals WHERE neutered = true;
+SELECT name from animals WHERE escape_attempts < 3;
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = true;
+SELECT *
+FROM animals
+WHERE NOT name = 'Gabumon';
+SELECT * from animals WHERE weight_kg between 10.4 and 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+
+
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+COMMIT;
+SELECT * FROM animals;
+
+
+
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+
+
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT savepoint1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO savepoint1;
+COMMIT;
```

Comment:
- [ ] As mentioned by the previous reviewer, based on the project requirement as shown below, you are expected that after the rollback, you `Update all animals' weights that are negative to be their weight multiplied by -1.` and then commit. 

**Project Requirement** 
--- | 
![image](https://user-images.githubusercontent.com/84156714/225571131-25eca2a5-6c9f-49b6-bc75-04f33805e9b7.png) 
```sql
BEGIN;
DELETE FROM animals WHERE date_of_birth > '2022-01-01';
SAVEPOINT savepoint1;
UPDATE animals SET weight_kg = weight_kg * -1;
ROLLBACK TO savepoint1;
UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; -- This is the missing query
COMMIT;
``